src/modules/bulletproofs/rangeproof_uncompressed_impl.h

```diff
@@ -402,4 +402,142 @@ static int secp256k1_bulletproofs_rangeproof_uncompressed_prove_step3_impl(
     return 1;
 }
 
+typedef struct {
+    const secp256k1_ge *t1p;
+    const secp256k1_ge *t2p;
+    const secp256k1_ge *asset_genp;
+    const secp256k1_ge *commitp;
+    secp256k1_scalar neg_t;
```

Comment:
I checked with both vim and emacs and neither has the behavior. It's unfortunate that Github is broken in this way but I don't think we should change our code because of it.